src/components/Messages/PreviewList.tsx

```diff
@@ -32,7 +32,7 @@ const PreviewList: FC<Props> = ({ className }) => {
   const [showSearchModal, setShowSearchModal] = useState(false);
   const { authenticating, loading, messages, profiles, profilesError } = useMessagePreviews();
 
-  if (!currentProfile || !isFeatureEnabled('messages', currentProfile.id)) {
+  if (!isFeatureEnabled('messages', currentProfile?.id)) {
```

Comment:
Same here -- I think we still want to 404 here if current profile is null even after the feature flag is stripped so I think the first check might be safer.